test/e2e_node/topology_manager_test.go

```diff
@@ -297,19 +301,27 @@ func findSRIOVResource(node *v1.Node) (string, int64) {
 	return "", 0
 }
 
+type testEnvInfo struct {
+	numaNodes         int
+	sriovResourceName string
+	policy            string
+}
+
 func validatePodAlignment(f *framework.Framework, pod *v1.Pod, envInfo *testEnvInfo) {
 	for _, cnt := range pod.Spec.Containers {
 		ginkgo.By(fmt.Sprintf("validating the container %s on Gu pod %s", cnt.Name, pod.Name))
 
 		logs, err := e2epod.GetPodLogs(f.ClientSet, f.Namespace.Name, pod.Name, cnt.Name)
 		framework.ExpectNoError(err, "expected log not found in container [%s] of pod [%s]", cnt.Name, pod.Name)
-
 		framework.Logf("got pod logs: %v", logs)
-		numaRes, err := checkNUMAAlignment(f, pod, &cnt, logs, envInfo)
-		framework.ExpectNoError(err, "NUMA Alignment check failed for [%s] of pod [%s]", cnt.Name, pod.Name)
-		if numaRes != nil {
-			framework.Logf("NUMA resources for %s/%s: %s", pod.Name, cnt.Name, numaRes.String())
+
+		cntFileRead := func(podName, cntName, path string) string {
```

Comment:
Why define this function here and plumb it through?  Since this is the only call site, it seems like it just adds complexity.